app/scripts/components/common/page-header/logo.tsx

```diff
@@ -74,7 +74,7 @@ const Brand = styled.div`
   }
 `;
 
-const PageTitleSecLink = styled(Link)`
+const PageTitleSecLink = styled.a`
```

Comment:
This will make the links on the nav plain a tags, which we don't want. How about we take a similar approach that `ElementInteractive` component is taking,(https://github.com/NASA-IMPACT/veda-ui/blob/main/app/scripts/components/common/card/index.tsx#L307)  passing the linkElement as `as` so the styled component can be whatever props says.

Reply:
oh, yep thats what I did actually => https://github.com/NASA-IMPACT/veda-ui/blob/06b3a016226b2f22a95fa9ab27d8a673b7525a65/app/scripts/components/common/page-header/logo.tsx#L112. I had to switch it to an `a` tag because we cant import `Link` from `react-router-dom` at this level so its moved up!

Reply:
Oh I see, sorry for missing that part!